CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 0.15.1 (September 10, 2020)
```

Comment:
`0.15.1` is already tagged on a different branch: https://github.com/tendermint/go-amino/blob/release/v0.15.1/CHANGELOG.md. I think we should put these changes in `0.16.0`. It appears that there are also some changes in `0.15.1` that haven't been merged back into master. @marbar3778 you did the last point release. Are you able to give us some guidance on doing a new release here?

Reply:
I believe the best approach would be to branch off of the most recent release branch or tag. It is unclear if master is working correctly, I vaguely remember there being a bug or two on master that were never fixed.

Reply:
Done. There's a changelog entry indeed for 0.15.1, but only in this release branch. I added a 0.16.0 one.

Reply:
Sorry for not being descriptive. Could you create a branch from release/0.15.1 like rc0/0.16 then put this PR against that?

Reply:
Ah ok, understood.